$NetBSD: patch-apps_tqsl_cpp,v 1.6 2018/03/17 02:45:05 mef Exp $

(1)
Avoid following error (interim, only valid with db5)
/usr/pkgsrc/wip/trustedQSL/work/tqsl-2.0/apps/tqsl.cpp:1133:36: error: 'DB_VERSION_STRING' was not declared in this scope
gmake[2]: *** [apps/CMakeFiles/tqsl.dir/tqsl.cpp.o] Error 1
gmake[1]: *** [apps/CMakeFiles/tqsl.dir/all] Error 2

(2)
tqsl-2.0.1/apps/tqsl.cpp:2963:10: error: #pragma GCC diagnostic not allowed inside functions
tqsl-2.0.1/apps/tqsl.cpp:2967:10: error: #pragma GCC diagnostic not allowed inside functions

(3) Trial code for wxGTK30

(4) Avoid error with wxGTK30
 error: cannot pass objects of non-trivially-copyable type 'class wxCStrData' through '...'

--- apps/tqsl.cpp.orig	2016-03-12 14:49:01.000000000 +0900
+++ apps/tqsl.cpp	2016-03-16 22:49:15.000000000 +0900
@@ -64,7 +64,7 @@
 #endif
 #include <zlib.h>
 #include <openssl/opensslv.h> // only for version info!
-#include <db.h> //only for version info!
+#include <db5/db.h> //only for version info!
 
 #include <iostream>
 #include <fstream>
